src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -193,6 +198,7 @@ export const exchangeStitchingEnvironment = ({
               }),
               // how to process the data result at path
               (result) => {
+                console.log("got result", { result })
```

Comment:
this piece is pretty key - after `delegateToSchema` I consistently logged `result: null` here when using mock functions like:
```
Me: () => {
           conversation: (_root, _params) => {		         //   console.log("Me mock")
            return {
              conversation: () => {
                console.log("conversation mock")
                return {
                  items: [
                    {
                      item: { __typename: "Artwork", title: "Conversation Art" },
                    },
                  ],
                }
              },
            }
          },
```
and
```
Query: () => ({
           me: () => ({
             conversation: {
               items: [
                  {
                    item: { __typename: "Artwork", title: "Conversation Art" },
                  },
                ],
              },
            }),
```